ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -224,10 +229,24 @@ func resourceIBMPICloudConnectionCreate(ctx context.Context, d *schema.ResourceD
 	}
 
 	client := st.NewIBMPICloudConnectionClient(ctx, sess, cloudInstanceID)
+	retryCount := 2
 	cloudConnection, cloudConnectionJob, err := client.Create(body)
 	if err != nil {
-		log.Printf("[DEBUG] create cloud connection failed %v", err)
-		return diag.FromErr(err)
+		for count := retryCount; count >= 0; count-- {
```

Comment:
How many times do you want to retry?
If it is 3 as expected then use `count > 0` here. Setting `retryCount := 2` and running loop 3 times does not make sense.